test/parallel/test-http-dump-req-when-res-ends.js

```diff
@@ -14,19 +17,14 @@ const server = http.createServer(function(req, res) {
     }, 1));
   }));
 
-  // end is not called as we are just exhausting
-  // the in-memory buffer
-  req.on('end', common.mustNotCall);
-
-  // this 'data' handler will be removed when dumped
-  req.on('data', common.mustNotCall);
```

Comment:
Can we still check that `_dump()` removes all data listeners?

https://github.com/nodejs/node/blob/354849eeb595ac364337e564669de4c77aa4c1af/lib/_http_incoming.js#L299

maybe adding an assertion for `listenerCount('data')` in the `'resume'` handler?

Reply:
Added